src/components/userCard.tsx

```diff
@@ -0,0 +1,27 @@
+import projectsCardIcon from "../assets/icons/projectsCardIcon.svg";
+import { UserCardProps } from "@/interfaces/userInterface";
```

Comment:
This interface seems to be particular to this component. I would suggest removing it from the shared interfaces folder, and implementing it directly here, and instead importing the User interface.

Reply:
i prefer to leave separated because i might use elsewhere, and its going to be the same atributes, keeping the code inside the userCard cleaner

Reply:
I agree with having a separate interface for User. What I'm suggesting is having the particular interface UserCardProps being implemented inside of this component, since it is particular to this one and not generic enough.

Reply:
ok, i deleted the interface because i'm not using for anything else right now, i put the interface inside the component